catena4801_simple/README.md

```diff
@@ -0,0 +1,312 @@
+
+It's easy to run, provided you're on Windows, macOS, or Linux, and provided you have `git` installed. We tested on Windows with git bash from https://git-scm.org, on macOS 10.11.3 with the git and bash shipped by Apple, and on Ubuntu 16.0.4 LTS (64-bit) with the built-in bash and git from `apt-get install git`.
+
+```console
+$ cd Catena-Sketches/catena4801_simple
+$ ../git-boot.sh
+Cloning into 'Catena-Arduino-Platform'...
+remote: Counting objects: 1201, done.
+remote: Compressing objects: 100% (36/36), done.
+remote: Total 1201 (delta 27), reused 24 (delta 14), pack-reused 1151
+Receiving objects: 100% (1201/1201), 275.99 KiB | 0 bytes/s, done.
+Resolving deltas: 100% (900/900), done.
+...
+
+==== Summary =====
+No repos with errors
+No repos skipped.
+*** no repos were pulled ***
+Repos downloaded:      Catena-Arduino-Platform arduino-lorawan Catena-mcciadk arduino-lmic Adafruit_FRAM_I2C Modbus-for-Arduino
+```
+
+It has a number of advanced options; use `../git-boot.sh -h` to get help, or look at the source code [here](https://github.com/mcci-catena/Catena-Sketches/blob/master/git-boot.sh).
+
+**Beware of issue #18**.  If you happen to already have libraries installed with the same names as any of the libraries in `git-repos.dat`, `git-boot.sh` will silently use the versions of the library that you already have installed. (We hope to soon fix this to at least tell you that you have a problem.)
+
+#### List of required libraries
+
+This sketch depends on the following libraries.
+
+*  https://github.com/mcci-catena/Catena-Arduino-Platform
+*  https://github.com/mcci-catena/arduino-lorawan
+*  https://github.com/mcci-catena/Catena-mcciadk
+*  https://github.com/mcci-catena/arduino-lmic
+*  https://github.com/mcci-catena/Adafruit_FRAM_I2C
+*  https://github.com/mcci-catena/Modbus-for-Arduino
+
+### Build and Download
+
+Shutdown the Arduino IDE and restart it, just in case.
+
+Ensure selected board is 'MCCI Catena 4801' (in the GUI, check that `Tools`>`Board "..."` says `"MCCI Catena 4801"`.
+
+In the IDE, use File>Open to load the `catena4801_simple.ino` sketch. (Remember, in step 1 you cloned `Catena-Sketches` -- find that, and navigate to `{somewhere}/Catena-Sketches/catena4801_simple/`)
+
+Follow normal Arduino IDE procedures to build the sketch: `Sketch`>`Verify/Compile`. If there are no errors, go to the next step.
+
+### Configure Modbus parameters
+
+To communicate with the Modbus slave, it is necessary to modify the Modbus slave parameter in the `setup_modbus` function as it matches the device parameters.
+
+```
+/* Modbus slave parameters */
+u8addr = 65;			// current target device: 01
+u8fct = 03;			// Function code
+u16RegAdd = 00;			// start address in device
+u16CoilsNo = 13;		// number of elements (coils or registers) to read
+host.begin(&mySerial, 9600);	// baud-rate at 9600
+```
+
+Connect your modbus slave�s A and B terminal to Catena 4801�s A and B terminal (JP3).
```

Comment:
There is special characters after slave and 4801.

Reply:
Made the required changes